tests/test_compute_funs.py

```diff
@@ -173,94 +177,94 @@ def test_magnetic_field_derivatives(DummyStellarator):
     B_rr = np.apply_along_axis(my_convolve, 0, data["B"], FD_COEF_2_4) / drho**2
 
     np.testing.assert_allclose(
-        data["B^theta_r"][3:-2],
-        B_sup_theta_r[3:-2],
+        data["B^theta_r"][4:-4],
```

Comment:
why make these further from the bdry? because you increased nrho so want to keep it at the same rho values roughly?

Reply:
My guess is same reason as https://github.com/PlasmaControl/DESC/pull/598#discussion_r1282271985